### DISCUSSION TITLE
Project Roadmap

### DISCUSSION THREAD

**Discussion by ksenginew** [TarunavBA/Social-Network#19] [General]
I want to discuss about the project roadmap here. Looks like the maintainer @TarunavBA is a beginner for python.
For basic-writing-and-formatting-syntax visit https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Comment by ksenginew** [comment]
Use this [template](https://github.com/othneildrew/Best-README-Template) for README

**Reply by TarunavBA** [reply]
Ok @ksenginew

**Comment by ksenginew** [comment]
This is how traditional Social sites work
![IMG_20211010_115145](https://user-images.githubusercontent.com/89186091/136685099-fce98a35-559a-4951-b4c2-9241a103b993.jpg)

**Reply by ksenginew** [reply]
I suggest 
![IMG_20211010_115208](https://user-images.githubusercontent.com/89186091/136685119-92112864-8e37-4244-bc37-3928b378e7f4.jpg)

**Reply by TarunavBA** [reply]
Sure but I don't know java/kotlin/c/c++ only I know html, css and python and obviously flask.

**Reply by TarunavBA** [reply]
If you know someone who could help us feel free to add them in this project. 😊

**Reply by ksenginew** [reply]
Looks like python is not suitable for very basic social network.

I already interested in creating a social network.
I would like to create a new organisation for this project. I already named my project as 'sless'(means server less).

I 'll create and manage a repo. You'll be a maintainer. Then you can manage contributors and other people. Then we can apply python and various programming languages.
What is your idea?

**Comment by TarunavBA** [comment]
Hey @ksenginew let me create a project board for our convenience.

**Reply by ksenginew** [reply]
Ok

**Reply by ksenginew** [reply]
But can I discuss about it here

**Reply by TarunavBA** [reply]
Ok 👌.

**Reply by ksenginew** [reply]
Looks like You're a newbie to python.
It's better if you can create a chat app. Then improve it to real social media.

**Reply by TarunavBA** [reply]
Ok. So first let's create the login page. I will file a feature request for this. Then we can get to work. 😊

**Reply by ksenginew** [reply]
R U in telegram. please join.

**Reply by TarunavBA** [reply]
Nope I don't have it and I'm not allowed to create one.

**Reply by ksenginew** [reply]
How fan we chat

**Reply by TarunavBA** [reply]
Like this only probably. 😁

**Reply by ksenginew** [reply]
https://gitter.im/ join I'm in https://gitter.im/Ksenginew

**Reply by ksenginew** [reply]
> https://gitter.im/ join I'm in https://gitter.im/Ksenginew

??

**Reply by TarunavBA** [reply]
Sorry I can't.

**Reply by ksenginew** [reply]
> Sorry I can't.

gitter is a website. It's a chat for developers.

**Comment by TarunavBA** [comment]
Then @ksenginew shall I rename it to `boo - a IRC built using flask`?

**Reply by ksenginew** [reply]
No. Name should be memorable short name.
Description should be long. 
Eg:
- name: boo
- a IRC built using flask

**Comment by TarunavBA** [comment]
Ok! Perfect.

~ 🤖
